### ISSUE TITLE
Suggestion: Unicode headings and configurable markdig with web.config 

### ISSUE THREAD

**Issue by DenisValeev** [jmalarcon/MIIS#30]
Hi José!

Take a look at the https://github.com/lunet-io/markdig/blob/master/src/Markdig/MarkdownExtensions.cs#L502

You can pass plus-separated list of extensions. Which can be configurable from web.config of your project.

Also, I had to patch MarkdownFile.cs to change the `markdig` behavior to process cyrillic headings correctly, instead of replacing them with `section`, `section-1` things as per this issue 95 @ markdig: https://github.com/lunet-io/markdig/issues/95

Consider adding these 2 lines to MarkdownFile.cs @ line 85: 
```
mdPipe.Extensions.Remove(mdPipe.Extensions.Find<AutoIdentifierExtension>());
mdPipe.UseAutoIdentifiers(AutoIdentifierOptions.AutoLink);
```
Ironically, such behavior cannot be configured with the mentioned plus-separated list of extensions.

**Comment by jmalarcon** [comment]
Hi Denis:

Thanks, I'll definitely take a look at it. Thanks. I'm now trying to finish version 3 with a lot of new features (specially for extending MIIS to one's will) and I'll try to include this too. Thanks!

**Comment by jmalarcon** [comment]
Hey Denis!

Just a line to tell you that I've implemented both suggestions in v 3.0, that is going to be ready very soon with **lots** of new features and much, much powerful.

Thanks!
